engine/runtime-fat-jar/src/main/java/module-info.java

```diff
@@ -1,8 +1,8 @@
 open module org.enso.runtime {
   requires java.base;
+  requires java.net.http;
+  // compiler shall not be need
```

Comment:
?

Reply:
This [commit](https://github.com/enso-org/enso/pull/9868/commits/751416d3351cf4dcd03b7e8bf1f2f1fe98306f50) is the answer. `engine/runtime` needed a transitive dependency on `java.compiler` because of using `String` constant from `interpreter-dsl` project.